src/osdc/Objecter.h

```diff
@@ -454,7 +467,8 @@ struct ObjectOperation {
   }
 
   void cmpext(uint64_t off, ceph::buffer::list&& cmp_bl, boost::system::error_code* ec,
-	      std::size_t* s) {
+	      uint64_t* s) {
+    if (s) *s = -1;
```

Comment:
It's almost certainly not needed. I can't think of a situation where the error handler wouldn't be called. I added it in defensively in case there were some situation I hadn't thought of. If it offends you I can remove it.

Reply:
Renamed.